### PR TITLE
fix(frontend): fix table option for internal table

### PR DESCRIPTION
**Pull request by Li0k**
I hereby agree to the terms of the [Singularity Data, Inc. Contributor License Agreement](https://gist.github.com/skyzh/0663682a70b0edde7ae991492f2314cb#file-s9y_cla).

## What's changed and what's your intention?

as title

Please explain **IN DETAIL** what the changes are in this PR and why they are needed:

- save properties in ctx
- fix table option for internal table

## Checklist

- ~[ ] I have written necessary rustdoc comments~
- ~[ ] I have added necessary unit tests and integration tests~
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

## Refer to a related PR or issue link (optional)
- https://github.com/singularity-data/risingwave/issues/4425

### CODE REVIEW

**Comment by Li0k** [comment]
some `Node` left
- TopN
- AppendOnlyTopN
- Lookup
- Arrangement

File: src/frontend/src/handler/create_mv.rs [hunk 1]
```diff
@@ -81,13 +77,13 @@ pub fn gen_create_mv_plan(
...
 
-    let mut plan_root = Planner::new(context).plan_query(bound)?;
+    let mut plan_root = Planner::new(context.clone()).plan_query(bound)?;
     plan_root.set_required_dist(RequiredDist::Any);
     let materialize = plan_root.gen_create_mv_plan(table_name)?;
     let mut table = materialize.table().to_prost(schema_id, database_id);
     let plan: PlanRef = materialize.into();
     table.owner = session.user_id();
-    table.properties = properties;
+    table.properties = context.inner().with_properties.clone();
```
Comment (st1page) [inline src/frontend/src/handler/create_mv.rs:86]: maybe we can add the properties in `StreamMaterialize::create`

Comment (Li0k) [reply src/frontend/src/handler/create_mv.rs]: fix it cc @st1page

**Review by st1page — APPROVED** [review]
LGTM

**Review by chenzl25 — APPROVED** [review]
LGTM

**Pull request merged**